### ISSUE TITLE
Dependency tree discrepancy

### ISSUE THREAD

**Issue by equals215** [nestjs/passport#928]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

While running `npm ci` in a fresh workflow environment the command is throwing this error :
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @nestjs/passport@8.2.1
npm ERR! Found: passport@0.6.0
npm ERR! node_modules/passport
npm ERR!   passport@"^0.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer passport@"^0.4.0 || ^0.5.0" from @nestjs/passport@8.2.1
npm ERR! node_modules/@nestjs/passport
npm ERR!   @nestjs/passport@"^8.2.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: passport@0.5.3
npm ERR! node_modules/passport
npm ERR!   peer passport@"^0.4.0 || ^0.5.0" from @nestjs/passport@8.2.1
npm ERR!   node_modules/@nestjs/passport
npm ERR!     @nestjs/passport@"^8.2.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```
However I currently have version 8.2.1 of @nestjs/passport installed - this was released on Feb 16 thus I can see in my local folder of node_modules/@nestjs/passport/package.json that it has the dependency "passport": "^0.4.0 || ^0.5.0 , as the terminal error implies

### Minimum reproduction code

https://gist.github.com/equals215/08a5c256c67965522b5bbe0065805174

### Steps to reproduce

1. `npm install`

### Expected behavior

No dependency conflict

### Package version

8.2.1

### Passport version

0.6.0

### NestJS version

8.4.6

### Node.js version

16.x.x

### In which operating systems have you tested?

- [X] macOS
- [ ] Windows
- [X] Linux

### Other

[This issue on SO](https://stackoverflow.com/questions/72509302/modifying-an-npm-package-locally-to-manually-fix-dependency-tree-issues/72509412)

**Comment by roryf** [comment]
Looks like #903 updated `peerDependencies` to include `^0.6.0`, any chance this could be release?

**Comment by kamilmysliwiec** [comment]
Fixed in v8.2.2
